packages/web/src/scss/components/UNSTABLE_Footer/index.html

```diff
@@ -0,0 +1,1123 @@
+{{#> web/layout/plain }}
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Basic Usage</h2>
+
+  <footer class="UNSTABLE_Footer">
```

Comment:
please add a wrapping stack, because it is part of the documenting component and it will be there in other technologies.

Reply:
Ok, I didn't think of that because there is only one child